lib/hl7/raw_message.ex

```diff
@@ -4,9 +4,16 @@ defmodule HL7.RawMessage do
   @moduledoc """
```

Comment:
@DarkMarmot  I wasn't sure if I should mark this one as deprecated, since it seems like `to_string` is still super useful. What do you think?

Reply:
Yes, let's keep this module deprecated but the doc on to_string is great thanks!